test/flows/plan_adoption_leave_flow_test.rb

```diff
@@ -0,0 +1,77 @@
+require "test_helper"
+require "support/flow_test_helper"
+
+class PlanAdoptionLeaveFlowTest < ActiveSupport::TestCase
+  include FlowTestHelper
+
+  setup { testing_flow PlanAdoptionLeaveFlow }
+
+  should "render a start page" do
+    assert_rendered_start_page
+  end
+
+  context "question: child_match_date?" do
+    setup { testing_node :child_match_date? }
+
+    should "render the question" do
+      assert_rendered_question
+    end
+
+    context "next_node" do
+      should "have a next node of child_arrival_date? for a response of 3 months ago" do
```

Comment:
```suggestion
      should "have a next node of child_arrival_date?" do
```

Since it doesn't matter what the response is to reach that node